src/main/java/com/znsio/teswiz/runner/BrowserDriverManager.java

```diff
@@ -213,7 +213,7 @@ private static ChromeOptions getChromeOptions(String forUserPersona, TestExecuti
 
         String browserVersion = getOverriddenStringValue(BROWSER_VERSION,
                 chromeConfiguration.getString("browserVersion"));
-        if (Runner.getPlatform().equals(Platform.web) && !browserVersion.equalsIgnoreCase("latest")) {
+        if ((Runner.getPlatform().equals(Platform.web) || Runner.getPlatform().equals(Platform.electron)) && !browserVersion.equalsIgnoreCase("latest")) {
```

Comment:
why are we changing this? This class is supposed to be only for web execution

Reply:
There is possibility that user is not setting browser version for electron